keyboards/ymdk/ymd09/info.json

```diff
@@ -10,19 +10,55 @@
     },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgb_matrix": true,
+        "lto": true,
+    },
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "cols": ["D6", "D2", "D1"],
+        "rows": ["F5", "F4", "F1"]
+    },
+    "bootmagic": {
+        "matrix": [0, 1] // Hold 8 on start up to clear EEPROM and boot into bootloader mode
+    },
```

Comment:
Why this change? That wasn't the prior behaviour nor should it be changed to something non-standard.

Reply:
Because this macropad does not have default bootmagic key at all. I can change readme file according to this change. The key can be changed to [0,0] to be more standart-like.

Reply:
> The key can be changed to [0,0] to be more standart-like.

This should be done, random configuration just makes it more complicated for end users.

The default is `0,0`, so adding that is pointless. It should just rely on the defaults, and the current config removed.